src/ensembl/src/header/headerActions.ts

```diff
@@ -1,25 +1,48 @@
 import { createAction } from 'typesafe-actions';
 
-import { getHeaderAnalyticsObject } from 'src/analyticsHelper';
+import buildAnalyticsObject from 'src/analyticsHelper';
 
 export const toggleAccount = createAction(
   'header/toggle-account',
   (resolve) => {
-    return () => resolve(undefined, getHeaderAnalyticsObject('Navigation'));
+    return () =>
+      resolve(
+        undefined,
+        buildAnalyticsObject({
+          category: 'Header',
+          action: 'toggle-account',
+          label: 'Navigation'
+        })
+      );
   }
 );
 
 export const toggleLaunchbar = createAction(
   'header/toggle-launchbar',
   (resolve) => {
-    return () => resolve(undefined, getHeaderAnalyticsObject('Navigation'));
+    return () =>
+      resolve(
+        undefined,
+        buildAnalyticsObject({
+          category: 'Header',
+          action: 'toggle-launchbar',
+          label: 'Navigation'
+        })
+      );
   }
 );
 
 export const changeCurrentApp = createAction(
   'header/change-current-app',
   (resolve) => {
     return (currentApp: string) =>
-      resolve(currentApp, getHeaderAnalyticsObject('Navigation'));
+      resolve(
+        currentApp,
+        buildAnalyticsObject({
+          category: 'Header',
+          action: 'change-current-app',
+          label: currentApp
+        })
+      );
```

Comment:
Do we still want to track the three actions in this file?

Reply:
Removed these.

Reply:
Yes, the URL changes are tracked.